vit/command_bar.py

```diff
@@ -59,8 +45,20 @@ def keypress(self, size, key):
             return None
         elif key in self.readline.keys():
             return self.readline.keypress(key)
+        elif key in ('backspace',) and self.abort_backspace and not self.get_edit_text():
```

Comment:
I"m wondering if breaking this line off into a separate ```abort_on_backspace()``` method would be cleaner? I think it might make the code a bit more readable there.

Reply:
Agreed, but I would call it `is_aborting_backspace(key)` if the function contains all three conditions in the line you quoted.

Reply:
OK, reasonable, go ahead with that name you suggested.